src/stirling/source_connectors/perf_profiler/notes.md

```diff
@@ -0,0 +1,209 @@
+# Stirling profiler notes
+
+### Perf profiler organization
+TODO(jps): describe the various source code files and their respective purposes.
+
+### Symbolization notes
+TODO(jps): cover symbolization for compiled & Java and symbol caching.
+
+### Missing stack frames
+While investigating issue [#719](https://github.com/pixie-io/pixie/issues/719), we noticed that infrequently (less than 0.5% of the time, e.g.) we observed a stack trace with a missing stack frame. Fixing the test case to accommodate the issue was easy, but not informative. Here, we document our investigation into the root cause of the issue.
+
```

Comment:
What do you think about adding the dot visualization of the missing stack frame here? I think it helps to ease into the explanation so that it's very clear what the issue was.

Reply:
I added the call graph rendered by pprof (my current opinion is that pprof provides a better cleaner visualization).